### DISCUSSION TITLE
Problems with GNOME Device Security (GNOME Settings)

### DISCUSSION THREAD

**Discussion by Odi-3** [fwupd/fwupd#6966] [Q&A]
Unfortunately, my device can't get "secure device" status in GNOME Device Security (part of GNOME Settings). Trying to print the report, it seems that some checks did not pass; this happens despite the fact that the hardware is relatively new and the firmware is updated to the latest version.
You can find more details about the device and the report [here](https://discourse.gnome.org/t/how-to-get-the-status-of-safe-device-in-device-security-settings/20082/3?u=odi-3).
I would be grateful if you could provide me with clarification and possibly guidance on how to resolve it.

Thank you very much.

**Comment by hughsie** [comment]
You need to talk to Acer -- that's a pretty terrible result for a new laptop I'm afraid.

**Reply by Odi-3** [reply]
Thank you for your feedback.
I doubt they would provide me support for a "problem" on Linux. I don't think the firmware doesn't support these features or that it hasn't been equipped with any security patches. I admit I don't know how fwupd collects that information but I rather think it's a communication issue with UEFI. It is unlikely that after four firmware updates (the latest of which was last November) such important features could be missing.
Let me know what you think. Does my reasoning make sense?

**Reply by hughsie** [reply]
It not Linux specific, it's the same firmware on Windows too. I think Acer is missing some very important CSME updates and hasn't set up the platform properly. I don't think security is hugely important to them to be honest.

**Reply by Odi-3** [reply]
Yes, it was to say that right now I only use Linux on the laptop because I think it's the best choice. In any case, I will try to ask them for clarification. In case they give me information that may be important to other users, I will report it here.
